### PR TITLE
NEWS: initial version of NEWS for v240

### PR DESCRIPTION
**Pull request by poettering**
Needs lots of updates still, but let's get the party started.

### CODE REVIEW

**Comment by poettering** [comment]
as usual, would be nicer to get follow-up PRs then fix suggestions right here.

**Comment by poettering** [comment]
OK, let's merge this. Please submit any fixes as follow-up PRs.

File: NEWS [hunk 1]
```diff
@@ -88,16 +88,342 @@ CHANGES WITH 240 in spe:
...
+          recognizable, too. A new recognizable is also added now whenever a
+          unit enters the "dead" state, on success.
+
+        * systemd-run gained a new switch --working-directory= for configuring
+          the working directory of the service to start. A shortcut -d is
+          equivalent, setting the working directory of the service to the
+          current working directory of the invoking program. The new --shell
+          (or just -S) option has been added for invoking the $SHELL of the
+          caller as a service, and implies --pty --same-dir --wait --collect
+          --service-type=exec. Or in other words, "systemd-run -S" is not the
```
Comment (gdamjan) [inline NEWS:341]:
```suggestion
          --service-type=exec. Or in other words, "systemd-run -S" is now the
```

File: NEWS [hunk 2]
```diff
@@ -88,16 +88,342 @@ CHANGES WITH 240 in spe:
...
+          quickest way to quickly get an interactive in a fully clean and
```
Comment (gdamjan) [inline NEWS:342]:
```suggestion
          quickest way to get an interactive in a fully clean and
```

**Comment by gdamjan** [comment]
> as usual, would be nicer to get follow-up PRs then fix suggestions right here.

oops, sorry didn't see this before clicking on those suggestions

**Pull request merged**